src/SFA.DAS.Commitments.Support.SubSite/Controllers/StatusController.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using System.Reflection;
+using System.Web;
+using System.Web.Http;
+using SFA.DAS.Commitments.Support.SubSite.Models;
+using SFA.DAS.Support.Shared;
+
+namespace SFA.DAS.Commitments.Support.SubSite.Controllers
+{
+    [RoutePrefix("api/status")]
+    public class StatusController : ApiController
+    {
+        [AllowAnonymous]
+        public IHttpActionResult Get()
+        {
+            return Ok(new ServiceStatusViewModel
+            {
+                ServiceName = AddServiceName(),
+                ServiceVersion = AddServiceVersion(),
+                ServiceTime = AddServerTime(),
+                Request = AddRequestContext()
+            });
+        }
+
+        private string AddServiceVersion()
+        {
+            try
+            {
+                return Assembly.GetExecutingAssembly().Version();
+            }
+            catch
+            {
+                return "Unknown";
+            }
+        }
+
+        private string AddRequestContext()
+        {
+            try
+            {
+                return $" {HttpContext.Current.Request.HttpMethod}: {HttpContext.Current.Request.RawUrl}";
+            }
+            catch
+            {
+                return "Unknown";
+            }
+        }
+
+        private DateTimeOffset AddServerTime()
+        {
+            return DateTimeOffset.UtcNow;
+        }
+
+        private string AddServiceName()
+        {
+            try
+            {
+                return "SFA DAS Commmitments Support Site";
```

Comment:
I think we can safely assume that this line won't throw, so we can remove the try..catch block.
Or even better, as it's a private method, we could remove the whole method and replace it with a const string.